docs/config-fcos-v1_5-exp.md

```diff
@@ -206,7 +206,8 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
     * **_threshold_** (int): sets the minimum number of pieces required to decrypt the device. Default is 1.
   * **_mirror_** (object): describes mirroring of the boot disk for fault tolerance.
     * **_devices_** (list of strings): the list of whole-disk devices (not partitions) to include in the disk array, referenced by their absolute path. At least two devices must be specified.
-* **_extensions_** (list of strings): a list of packages to layer on top of the OS.
+* **_extensions_** (list of objects): the list of packages to layer on top of the OS.
+  * **name** (string): the name of the package.
```

Comment:
This is correctly marked required here, but we're not actually enforcing that.  It's not _strictly_ necessary to do so (and it's unlikely that a user will forget the field until we have other fields in the struct) but would provide better UX than failing at runtime.  Could we add validation in `validate.go`, and a test for the validation?